crmsh/utils.py

```diff
@@ -2166,4 +2176,26 @@ def debug_timestamp():
     return datetime.datetime.now().strftime('%Y/%m/%d %H:%M:%S')
 
 
+def get_member_iplist():
+    rc, out, err= get_stdout_stderr("corosync-cmapctl -b runtime.totem.pg.mrp.srp.members")
+    if rc != 0:
+        common_debug(err)
+        return None
+
+    ip_list = []
+    for line in out.split('\n'):
+        match = re.search(r'ip\((.*?)\)', line)
+        if match:
+            ip_list.append(match.group(1))
+    return ip_list
+
+
+def test_ssh_need_passwd(hosts):
```

Comment:
@liangxin1300 I would avoid to use `test` prefix on a real function as it can cause confusion with unit tests. I would suggest here `check_ssh_passwd_need`.